src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.exception.FruitStoreException;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final String FIRST_LINE = "b,banana,21";
+    private static final String FILE_NAME_NOT_OK = "input_any.csv";
+    private static final String FILE_NAME_OK = "input.csv";
+    private static final String FILE_NAME_EMPTY_NOT_OK = "input_empty.csv";
+    private static ReaderServiceImpl readerService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test(expected = FruitStoreException.class)
+    public void readFile_FileNotFound_notOk() {
+        readerService.readFileToList(FILE_NAME_NOT_OK);
+        fail("Expected " + FruitStoreException.class.getName()
+                + " to be thrown for not existing file, but it wasn't");
```

Comment:
u should recieve fail when `FruitStoreException` thrown, this fail is useless. remove everywhere

Reply:
took away